### PR TITLE
Fix/vmih msgpack

### PR DESCRIPTION
**Pull request by vmihailenco**
This PR updates import path gopkg.in/vmihailenco/msgpack.v2 -> github.com/vmihailenco/msgpack (v4)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -49,24 +49,4 @@ clean:
...
-	go get -u github.com/tinylib/msgp/msgp
-	go get -u github.com/ugorji/go/codec
-	go get -u gopkg.in/mgo.v2/bson
-	go get -u gopkg.in/vmihailenco/msgpack.v2
-	go get -u github.com/golang/protobuf/proto
-	go get -u github.com/hprose/hprose-go/io
-	go get -u github.com/pascaldekloe/colfer/cmd/colf
-	go get -u github.com/calmh/xdr
-	go get -u github.com/niubaoshu/gotiny
+	go get -u
```
Comment (vmihailenco) [inline Makefile]: Go Modules should automatically install everything. `go get -u` was left to update all dependencies to the latest version.

**Comment by alecthomas** [comment]
> Tests can't be run at the moment because github.com/DeDiS/protobuf uses http://go.dedis.ch/protobuf which is not available at the moment (at least here), but I guess it is not my decision to remove it or not from the suite. But if you remove github.com/DeDiS/protobuf tests/benchmarks run as expected.

You might be able to use a `replace` directive in `go.mod` to work around this for now.

**Comment by vmihailenco** [comment]
It looks like replace directive expects a local filepath, not github import. Anyway this PR does not break http://go.dedis.ch/protobuf so no reason it should fix it :)

**Comment by vmihailenco** [comment]
Actually it loooks like (my) go mod does not work with http://go.dedis.ch/protobuf so I've just removed the commit that adds go.mod file. So this PR is only about updating import path.

**Comment by alecthomas** [comment]
Thanks!

**Pull request merged**